libbeat/scripts/Makefile

```diff
@@ -265,11 +266,11 @@ update: python-env collect
 	@echo "Updating generated files for ${BEAT_NAME}"
 
 	@# Update config files.
-	@cat _meta/beat.yml ${ES_BEATS}/libbeat/_meta/config.yml | sed -e "s/beatname/${BEAT_NAME}/g" > ${BEAT_NAME}.yml
+	@cat _meta/beat.yml ${ES_BEATS}/libbeat/_meta/config.yml | sed -e "s/beatname/${BEAT_NAME}/g;s/index-prefix/${BEAT_INDEX_PREFIX}/g" > ${BEAT_NAME}.yml
```

Comment:
I would prefer to call it `beat-index-prefix` to have it in sync with the ENV variable. Also I think the name is less likely to show up by accident somewhere else in the file.